data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,830 @@
+			action
+				set "license: Avgi Civilian"
+			`	"Conclusion. For now, we will leave you with some time and space to think. The former captain of the ship that brought the other humans, one Darius Calwell, is currently advising construction on the cylinder habitat in this system. You will find him on Aktina Cylinder, if you wish to speak with him."`
+			`	You are dismissed, and on the way out you are given a black hairpin-sized translator device, a smaller cousin of the one the Avgi have been using to communicate with you.`
+			action
+				set "language: Avgi"
+
+mission "Avgi: Humans From the Deep"
+	name "Humans among the Avgi"
+	description "Visit the <destination>, where there is another human, Darius Calwell, stranded in Avgi space."
+	source "Peripheria"
+	landing
+	destination "Aktina Cylinder"
+	to offer
+		has "Avgi: First Contact: done"
+
+
+mission "Avgi: Defend Ensemble 1"
+	name "Avgi Strike Force"
+	description "Meet up with the Avgi strike force at <destination>."
+	source "Aktina Cylinder"
+	landing
+	passengers 1
+	blocked "You'll need a free bunk on the <ship> if you want to take Darius anywhere off this station."
+	destination "Napigeo Station"
+	to offer
+		has "Avgi: Humans From the Deep: done"
+	on offer
+		conversation
+			`From afar, Aktina station is a titanic skeleton, with graphene ribs and a metallic backbone. Reminiscent of the human concept of an O'Neil Cylinder, it will be able to house millions of Avgi once completed. For now though, its construction has been hampered by the constant demand for new warships and mining stations throughout the Tangled Shroud, to the point that many are skeptical of its projected completion time within another few years.`
+			`	Following the instructions of the station's control authority, you fly through the station's open frame to its small pressurized area, which houses the construction workers and maintenance crews. You've been told that word of your arrival has been sent ahead of you, so you aren't entirely surprised to see a middle-aged human waiting in the hangar, a look of pure astonishment on his face as your ship settles into a free hangar. You disembark from the <ship> to find Darius waiting right outside the hatch.`
+			choice
+				`	"Hello."`
+				`	"Hey there."`
+				`	"I come in peace."`
+			`	The man blinks in surprise, then extends a hand, which you shake. "So, another one of us here in Fairyland. I don't know if I should be glad to see another friendly face, or feel bad that another soul is stranded here along with us." He shakes his head. "Let's find somewhere to talk. If you don't mind telling me, how'd you end up here anyway?"`
+			choice
+				`	"I have a jump drive."`
```

Comment:
Kind of the point, because it goes to the same label, but your choice whether to put it in or not. Just being able to say one thing is my great though.